three.js/src/threex/threex-armarkercontrols.js

```diff
@@ -251,6 +255,8 @@ ARjs.MarkerControls.prototype._initArtoolkit = function(){
 		}else if( _this.parameters.type === 'barcode' ){
 			artoolkitMarkerId = _this.parameters.barcodeValue
 			arController.trackBarcodeMarkerId(artoolkitMarkerId, _this.parameters.size);
+		}else if( _this.parameters.type === 'nft' ){
+			console.log('nft test');
```

Comment:
Here we could import the artoolkit.worker.js into the Worker but it require a path,
example code:

```js
// init the worker
var worker = new Worker ('path/to/artoolkit.worker.js');
```